locator/tablets.cc

```diff
@@ -365,7 +365,23 @@ api::timestamp_type tablet_map::resize_phase() const {
     return _resize.phase;
 }
 
-
+table_load_stats& table_load_stats::operator+=(const table_load_stats& s) noexcept {
+    size_in_bytes = size_in_bytes + s.size_in_bytes;
+    split_ready_phase = std::min(split_ready_phase, s.split_ready_phase);
+    return *this;
+}
+
+load_stats& load_stats::operator+=(const load_stats& s) noexcept {
+    for (auto& [id, stats] : s.tables) {
+        if (!tables.contains(id)) {
+            // assign the first, so subsequent aggregations happen on actually retrieved data.
+            tables[id] = stats;
```

Comment:
You could default the timestamp to max_timestamp, and then the std::min() works for a default-constructed table_load_stats.